app/views/admin/pages/dashboard.html.erb

```diff
@@ -1,2 +1,3 @@
 Welcome in dashboard<br />
-<%= link_to 'Admin Kudos', admin_kudos_path %>
+<%= link_to 'Admin Kudos', admin_kudos_path %><br/>
+<%= link_to 'Admin Company Values', admin_company_values_path %>
```

Comment:
In my understanding, this link should be present in the navbar as well:
_There is a link in admin's navigation to go to company values path_